pyaerocom/aeroval/obsentry.py

```diff
@@ -146,67 +204,3 @@ def get_vert_code(self, var):
                 f"{self.SUPPORTED_VERT_CODES}."
             )
         return val
-
-    def check_cfg(self):
-        """Check that minimum required attributes are set and okay"""
-
-        if not self.is_superobs and not isinstance(self.obs_id, str | dict):
-            raise ValueError(
-                f"Invalid value for obs_id: {self.obs_id}. Need str or dict "
-                f"or specification of ids and variables via obs_compute_post"
-            )
-        if isinstance(self.obs_vars, str):
-            self.obs_vars = [self.obs_vars]
-        elif not isinstance(self.obs_vars, list):
-            raise ValueError(f"Invalid input for obs_vars. Need list or str, got: {self.obs_vars}")
-        ovt = self.obs_vert_type
-        if ovt is None:
-            raise ValueError(
-                f"obs_vert_type is not defined. Please specify "
-                f"using either of the available codes: {self.SUPPORTED_VERT_CODES}. "
-                f"It may be specified for all variables (as string) "
-                f"or per variable using a dict"
-            )
-        elif isinstance(ovt, str) and ovt not in self.SUPPORTED_VERT_CODES:
-            self.obs_vert_type = self._check_ovt(ovt)
-        elif isinstance(self.obs_vert_type, dict):
-            for var_name, val in self.obs_vert_type.items():
-                if val not in self.SUPPORTED_VERT_CODES:
-                    raise ValueError(
-                        f"Invalid value for obs_vert_type: {self.obs_vert_type} "
-                        f"(variable {var_name}). Supported codes are {self.SUPPORTED_VERT_CODES}."
-                    )
-        ovl = self.instr_vert_loc
-        if isinstance(ovl, str) and ovl not in self.SUPPORTED_VERT_LOCS:
-            raise AttributeError(
-                f"Invalid value for instr_vert_loc: {ovl} for {self.obs_id}. "
-                f"Please choose from: {self.SUPPORTED_VERT_LOCS}"
-            )
-
-    def _check_ovt(self, ovt):
-        """Check if obs_vert_type string is valid alias
-
-        Parameters
-        ----------
-        ovt : str
-            obs_vert_type string
-
-        Returns
-        -------
-        str
-            valid obs_vert_type
-
-        Raises
-        ------
-        ValueError
-            if `ovt` is invalid
-        """
-        if ovt in self.ALT_NAMES_VERT_CODES:
-            _ovt = self.ALT_NAMES_VERT_CODES[ovt]
-            logger.warning(f"Please use {_ovt} for obs_vert_code and not {ovt}")
-            return _ovt
-        valid = self.SUPPORTED_VERT_CODES + list(self.ALT_NAMES_VERT_CODES)
-        raise ValueError(
-            f"Invalid value for obs_vert_type: {self.obs_vert_type}. "
-            f"Supported codes are {valid}."
-        )
```

Comment:
Have you made any validator that does the same as this function?

Reply:
Implemented now :+1: